examples/official/cifar10_cnn_pytorch/const.yaml

```diff
@@ -11,6 +13,7 @@ hyperparameters:
 searcher:
   name: single
   metric: validation_error
-  max_steps: 50
+  max_steps: 500
```

Comment:
we may want to add how many epochs this equates to - similar to the mnist.

Reply:
added